src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/domene/BehandlingRepository.kt

```diff
@@ -184,4 +184,44 @@ interface BehandlingRepository : JpaRepository<Behandling, Long> {
         nativeQuery = true,
     )
     fun finnBehandlingerSomSkalRekjøresLovendringForFremtidigOpphør(): List<Long>
+
+    @Query(
```

Comment:
Man kan vel kanskje filterere bort de med fremtidig opphør, siden de i følge magnus har 12% sjanse for å være med i queryen som Uy & magnus jobber med